### ISSUE TITLE
@angular/language-service ability to quiet errors...

### ISSUE THREAD

**Issue by born2net** [angular/angular#13726]
```
[ ] bug report => search github for a similar issue or PR before submitting
[x ] feature request
[ ] support request => Please do not submit support request here, instead see https://github.com/angular/angular/blob/master/CONTRIBUTING.md#question
```

**Current behavior**
```@angular/language-service ability to quiet errors... ```
will report every issue and it makes it very hard to adopt at this late stage

**Expected behavior**
I would love to be able to quiet errors in templates via something like:
```<!-- NG-IGNORE-ERROR-->``` before the line and 
```<!-- NG-IGNORE-PAGE-ERROR-->``` to ignore entire template / page errors

**Minimal reproduction of the problem with instructions**
none

**What is the motivation / use case for changing the behavior?**
it will make it very hard to move to this awesome project without a ability to quiet errors and slowly enable

**Please tell us about your environment:**
Windows 10 x64

* **Angular version:** 2.0.X
latest all ng2 / ts 2.1

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Chrome
 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
TS

**Comment by born2net** [comment]
tx for the label and would love to see this in next version so we can begin testing this awesome service, right now just too many errors to begin deploying so we wan to adopt only on new files we are creating.. at first, tx

**Comment by born2net** [comment]
bump!

**Comment by renyajie** [comment]
233

**Comment by xmeng1** [comment]
+1 
The Webstorm always tell me the component or directive need to "Considering to add it to a NgModule declaration"

**Comment by devoto13** [comment]
@xmeng1 This was a [fix](https://github.com/angular/angular/pull/21337) for this bug in Angular 5.2.1. Updating fixed WebStorm reporting every component as not included in the module.

**Comment by Tom4U** [comment]
I'm using latest 5.2.9 and still facing this issue. It's really annoying in Webstorm as it shows up as error.
Any idea why it's being the same behaviour, though it should be fixed?

**Comment by ovangle** [comment]
The "consider adding it to a NgModule" error is incredibly annoying when attempting to write mock components for unit tests. They're not supposed to be in a module (apart from the module I configure using `TestBed` and, they never will be.

If that was the only one that was supressable I'd be a happy man.

**Comment by PhilippMDoerner** [comment]
I also agree that being able to quiet errors that the Language Service incorrectly thinks are there would be highly appreciated.

Right now for example it can not recognize functions that were added to a component via decorator, which makes it very annoying to use when you added some utility functions to your component via decorator and want to use them in the template.

**Comment by literalpie** [comment]
A use-case we have for this is developing with Storybook. We often use the `moduleMetadata` decorator to declare a demo component to be used in a story. Because Angular Language Service does not understand how storybook works, it will throw errors when the component template is opened. (we can work around this issue by making a demoModule for the component, but we have a lot of existing components that don't do this)

Of course, it would be great if the language service could understand the storybook syntax, but I understand that is probably unrealistic.

The second best way to handle this would be to fail a little more "gracefully" and still try to make some functionality work even if the language service cannot figure out which module it belongs to (hopefully you don't need to understand the module to understand properties declared in the component class)

Lastly, if none of the above is possible, allowing exclusion of files would be helpful. I like the idea of comments similar to what linting rules commonly do.
